sdk/opa.go

```diff
@@ -422,3 +423,16 @@ func bundles(ctx context.Context, store storage.Store, txn storage.Transaction)
 	}
 	return bundles, nil
 }
+
+// Runtime returns the runtimes params of the OPA object
+func (opa *OPA) Runtime(ctx context.Context) (*runtime.Runtime, error) {
```

Comment:
I don't see a need for this function and I'd rather not extend the OPA SDK API if possible.